src/Tribe/Main.php

```diff
@@ -3019,6 +3019,15 @@ public static function deactivate( $network_deactivating ) {
 			add_action( 'shutdown', array( $deactivation, 'deactivate' ) );
 		}
 
+		/**
+		 * plugin uninstall callback
+		 * @see register_uninstall_hook()
+		 */
+		public static function uninstall() {
+			Tribe__Events__Integrations__Manager::instance()->load_freemius();
+			tribe( 'events.integrations.freemius' )->uninstall();
```

Comment:
This method doesn't exist in the Freemius class or ours.

Reply:
yeah, I wanted it to fatal, but it never gets to this point in the code :(